fili-core/src/main/java/com/yahoo/bard/webservice/table/ConcretePhysicalTable.java

```diff
@@ -26,19 +27,21 @@
      * @param timeGrain  time grain of the table
      * @param columns  The columns for this table
      * @param logicalToPhysicalColumnNames  Mappings from logical to physical names
+     * @param metadataService datasource metadata service containing availability data for the table
      */
     public ConcretePhysicalTable(
             @NotNull TableName name,
             @NotNull ZonedTimeGrain timeGrain,
-            @NotNull Iterable<Column> columns,
-            @NotNull Map<String, String> logicalToPhysicalColumnNames
+            @NotNull Set<Column> columns,
```

Comment:
I am making it a set because I do not yet to see any place requiring the behavior "Iteration order doesn't really matter, but we'll preserve it, so don't worry if it does.", if this behavior is not required, I prefer using a more standard approach of `Set`. Let me know if there is a case this is needed and I will change it back.